src/useLogger/useLogger.ts

```diff
@@ -0,0 +1,13 @@
+import { DependencyList } from 'react';
+import { useUpdateEffect, useMountEffect } from '..';
+
+export function useLogger(componentName: string, deps?: DependencyList, ...rest: any): void {
```

Comment:
Shouldn't dependencies be a replacement for `rest` argument?

Also I'd consider adding optional argument that will change the logging function, as it might come in handy in some situations and will simplify tests.

Reply:
Done